buildSrc/src/main/groovy/com/uber/okbuck/core/model/kotlin/KotlinLibTarget.groovy

```diff
@@ -21,4 +24,14 @@ class KotlinLibTarget extends JavaLibTarget {
     protected Set<File> getTestSrcDirs() {
         return (project.sourceSets.test.java.srcDirs as Set) + (project.sourceSets.test.kotlin.srcDirs as Set)
     }
+
+    @Override
+    Scope getApt() {
+        return Scope.from(project, ANNOTATION_PROCESSOR_CONFIGURATION_NAME)
+    }
+
+    @Override
+    Scope getTestApt() {
+        return Scope.from(project, ANNOTATION_PROCESSOR_CONFIGURATION_NAME)
```

Comment:
This should be testKapt?

Ideally you want to use the same constants the kotlin gradle plugin uses. Can you please check if they have the constants available publicly and switch to using those instead?

Reply:
They do not, searched inside kotlin-gradle-plugin project. Only place I've found kapt written was in [here](https://github.com/JetBrains/kotlin/blob/6838873a93f2fc33ccdbaea5788e08fc154ac2d1/gradle/project-schema.json) 😢

Reply:
They have it defined in https://github.com/JetBrains/kotlin/blob/1f81c0cdfe9d9438667094d01f2bd69f0d6fa780/libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/internal/Kapt3KotlinGradleSubplugin.kt#L79

but its internal.

You should be careful because there are variant specific kapt configurations as well for android projects which this pr does not capture correctly